src/vivarium_census_prl_synth_pop/tools/make_results.py

```diff
@@ -308,6 +309,8 @@ def load_data(raw_results_dir: Path, seed: str) -> Dict[str, pd.DataFrame]:
                 else:
                     df = pd.read_csv(file)
                 df["random_seed"] = file.name.split(".")[0].split("_")[-1]
+                if "age" in df.columns:
```

Comment:
The proper way to do this is to add a formatter to the age column that gets called by `formatter.format_columns()`